src/layer/AbstractSentinelHubV3Layer.ts

```diff
@@ -171,4 +171,43 @@ export class AbstractSentinelHubV3Layer extends AbstractLayer {
       headers: { 'Accept-CRS': 'EPSG:4326' },
     });
   }
+
+  protected fetchDates(
+    bbox: BBox,
+    fromTime: Date,
+    toTime: Date,
+    maxCloudCoverage?: number | null,
```

Comment:
Should we be explicit here that we are using a percentage value? Currently sometimes we are, and sometimes we are not...

Maybe we could:
- rename this to `maxCloudCoveragePercent`, or
- make a special type called `CloudCoveragePercent` and use it here instead of `number`?

Reply:
On the other hand, we are not consistent in the naming elsewhere either, so maybe let's make a separate MR which consolidates "cloud coverage" handling. Resolving.